src/unix/core.c

```diff
@@ -198,6 +198,19 @@ void uv__make_close_pending(uv_handle_t* handle) {
   handle->loop->closing_handles = handle;
 }
 
+int uv__getiovmax(void) {
+#if defined(IOV_MAX)
+  return IOV_MAX;
+#elif defined(_SC_IOV_MAX)
+  static int iovmax = -1;
+  if (iovmax == -1)
+    iovmax = sysconf(_SC_IOV_MAX);
```

Comment:
I don't think it's libuv's responsibility to do this. Not to mention all platforms may not support it. Last, this code has been in use in the streams implementation for ages, if we get more buffers than the maximum we will just do more writev syscalls.